fecgov/fec-cms
# Add SSF, lobbyist, and leadership PAC filters back to receipts/disbursement datatables

## Summary (required)

- Resolves #3054 
Add SSF, lobbyist registrant, and leadership PAC filters back to receipts/disbursement datatables

## Impacted areas of the application
List general components of the application that this PR will affect:

- Receipts/Individual contributions and disbursements datatables

## Screenshots

![Screen Shot 2019-07-30 at 1 52 24 PM](https://user-images.githubusercontent.com/31420082/62152946-8fae8900-b2d1-11e9-88b6-4d9e41e94ef1.png)

## Related PRs
List related PRs against other branches:

PR to remove SSFs from receipts: https://github.com/fecgov/fec-cms/pull/2941
PR to remove SSFs and leadership pacs from disbursements, leadership pacs from receipts: https://github.com/fecgov/fec-cms/pull/2979

## How to test

- Go to Receipts and Individual contributions datatables and make sure you can filter by SSFs, leadership PACs, and lobbyist/registrant PACs
   - http://localhost:8000/data/receipts/
   - http://localhost:8000/data/receipts/individual-contributions/
- Go to disbursement datatable and make sure you can filter by SSFs, leadership PACs, and lobbyist/registrant PACs
   - http://localhost:8000/data/disbursements/
- Ensure that other datatables with this `committee-types.jinja` filter is uneffected and still includes the SSFs in the PACS list.
   - http://localhost:8000/data/committees/
